packages/frontend/src/hooks/useCrabPosition/useCrabPosition.ts

```diff
@@ -107,11 +108,13 @@ export const useCrabPosition = (user: string) => {
 */
 export const useCrabPositionV2 = (user: string) => {
   const crabLoading = useAtomValue(crabLoadingAtomV2)
+  const [crabPositionRefetching, setCrabPositionRefetching] = useAtom(crabPositionRefetchingAtom)
```

Comment:
can we just use `useState` here to keep things simple? as far as I can see this state is not being used elsewhere.